sparse/coo/indexing.py

```diff
@@ -450,3 +450,36 @@ def _join_adjacent_pairs(starts_old, stops_old):  # pragma: no cover
     stops.append(stops_old[-1])
 
     return starts, stops
+
+
+def _normalize_axis(axis, ndim):
```

Comment:
Best to move this function into `utils`. `indexing` is `COO` specific and I'm pretty sure this will be needed elsewhere down the road.